test/spec/modules/biddoBidAdapter_spec.js

```diff
@@ -0,0 +1,172 @@
+import {expect} from 'chai';
+import {spec} from 'modules/invamiaBidAdapter.js';
```

Comment:
This should be testing the `biddoBidAdapter`.

Also, will this adapter always have the same logic as `invamia`? If so, would it be possible to make this an alias of `invamia` instead of a separate adapter?